packages/postcss-reduce-idents/src/lib/grid-template.js

```diff
@@ -20,23 +35,13 @@ module.exports = function () {
   /** @type {import('postcss').Declaration[]} */
   let declCache = [];
 
-  const gridChildProperties = new Set([
```

Comment:
Moving this outside of the function as there's no need to re-create it for every call.